reporting/impl/src/main/java/org/jboss/windup/reporting/rules/rendering/RenderGraphRuleProvider.java

```diff
@@ -17,13 +16,11 @@
 import org.ocpsoft.rewrite.context.EvaluationContext;
 
 /**
- *
  * This renders the graph itself to the output directory for debug purposes.
  *
  * @author <a href="mailto:jesse.sightler@gmail.com">Jesse Sightler</a>
- *
  */
-@Vetoed
+@Vetoed // When enabling, also delete the <exclude>**/viz.js</exlude> from POM.
```

Comment:
why the exclusion should be deleted, when you removed the file below? That comment doesn't make sense to me.